--- /_dist/deps/libco/Makefile	2020-03-12 00:02:37.744628626 +0100
+++ /_dist/deps/libco/Makefile	2020-03-12 00:00:15.212592603 +0100
@@ -3,7 +3,7 @@
 DESTDIR ?=
 PREFIX ?= /usr
 INCLUDEDIR ?= include
-LIBDIR ?= lib
+LIBDIR ?= lib/lxd
 MAJ = 0
 MIN = 0.1.0

@@ -20,9 +20,9 @@
 	install -d $(DESTDIR)$(PREFIX)/$(INCLUDEDIR)/
 	install -d $(DESTDIR)$(PREFIX)/$(LIBDIR)/
 	install -d $(DESTDIR)$(PREFIX)/$(LIBDIR)/pkgconfig/
-	install -m 0664 libco.so.$(MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)/libco.so.$(MIN)
-	install -m 0664 libco.pc $(DESTDIR)$(PREFIX)/$(LIBDIR)/pkgconfig/
+	install -m 0755 libco.so.$(MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)/libco.so.$(MIN)
+	install -m 0644 libco.pc $(DESTDIR)$(PREFIX)/$(LIBDIR)/pkgconfig/
 	ln -s libco.so.$(MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)/libco.so.$(MAJ)
 	ln -s libco.so.$(MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)/libco.so
-	install -m 0664 libco.a $(DESTDIR)$(PREFIX)/$(LIBDIR)/
-	install -m 0664 libco.h $(DESTDIR)$(PREFIX)/$(INCLUDEDIR)/
+	install -m 0644 libco.a $(DESTDIR)$(PREFIX)/$(LIBDIR)/
+	install -m 0644 libco.h $(DESTDIR)$(PREFIX)/$(INCLUDEDIR)/
